Source/CarthageKit/Xcode.swift

```diff
@@ -36,7 +36,7 @@ private func compilerVersionArguments(usingToolchain toolchain: String?) -> [Str
 private func parseSwiftVersionCommand(output: String?) -> String? {
 	guard
 		let output = output,
-		let regex = try? NSRegularExpression(pattern: "Apple Swift version ([^\\s]+) .*\\((.+)\\)", options: []),
+		let regex = try? NSRegularExpression(pattern: "Apple Swift version ([^\\s]+) .*\\((.[^\\)]+)\\)", options: []),
```

Comment:
Before the change this regex would match the second capture group differently in case of no new line:

`Apple Swift version 4.1.2 effective-3.3.2 (swiftlang-902.0.54 clang-902.0.39.2) -emit-object` -> `swiftlang-902.0.54 clang-902.0.39.2) -emit-object` (Garbage at the end)

But (notice new line char)

`Apple Swift version 4.1.2 effective-3.3.2 (swiftlang-902.0.54 clang-902.0.39.2) \n` ->  `swiftlang-902.0.54 clang-902.0.39.2

What we want is `swiftlang-902.0.54 clang-902.0.39.2`

The regex worked fine as it was here https://regex101.com/ but not with NSRegularExpression . Had to tell it to exclude `)`.